### PR TITLE
houdini: init at 16.0.633

### PR DESCRIPTION
**Pull request by canndrew**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by canndrew** [comment]
This is a build script for [Houdini](https://sidefx.com), a *proprietary* 3D animation program. It requires the user to download the installation file manually (using `requireFile`).

**Comment by canndrew** [comment]
Also, this depends on the undaemonize tool which [I've PR-ed separately](https://github.com/NixOS/nixpkgs/pull/27325).

File: pkgs/applications/misc/houdini/runtime.nix [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+in
+stdenv.mkDerivation rec {
+  version = "16.0.633";
+  name = "houdini-runtime-${version}";
+  src = requireFile rec {
+    name = "houdini-16.0.633-linux_x86_64_gcc4.8.tar.gz";
+    sha256 = "1laxncwgsr4hj53bn4pn9ibv3pkrpliwxlx0558wgnhq42js3wvl";
+    message = ''
+      This nix expression requires that ${name} is already part of the store.
+      Download it from SideFX's website and add it to the nix store with:
```
Comment (zimbatm) [inline pkgs/applications/misc/houdini/runtime.nix]: Is there any url you can point the user to?

File: pkgs/applications/misc/houdini/runtime.nix [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        echo "Patching ELF $BIN"
+        patchelf --set-interpreter "$INTERPRETER" "$BIN"
+      fi
+    done
+  '';
+  meta = {
+    description = "3D animation application software";
+    homepage = "https://sidefx.com";
+    license = stdenv.lib.licenses.unfree;
+    platforms = stdenv.lib.platforms.linux;
```
Comment (zimbatm) [inline pkgs/applications/misc/houdini/runtime.nix]: Are you willing to put yourself as the maintainer of the package?

**Comment by canndrew** [comment]
I've added fixes. Sorry about all the --force pushes.

**Comment by zimbatm** [comment]
@canndrew no need to be sorry :)

**Pull request merged**